frontend/ui/elements/screen_notification_menu_table.lua

```diff
@@ -0,0 +1,175 @@
+                    return isMaskEnabled(Notification.SOURCE_DISPATCHER)
+                end,
+                callback = function()
+                    toggleMask(Notification.SOURCE_DISPATCHER)
+                end,
+                },
+                {
+                text = _("Other sources"),
+                checked_func = function()
+                    return isMaskEnabled(Notification.SOURCE_OTHER)
+                end,
+                callback = function()
+                    toggleMask(Notification.SOURCE_OTHER)
+                end,
+                },
+            },
+        }
+    end
+    return ""
+end
+
+return {
+    text = _("Notification level"),
+    help_text = _([[KOReader may show notification popups at top of screen on various occasions.
+You can decide here which kind of notifications you want displayed.]]),
+    checked_func = function()
+        local value = G_reader_settings:readSetting("notification_sources_to_show_mask") or Notification.SOURCE_DEFAULT
+        return  value ~= 0
+    end,
+    sub_item_table = {
+        {
+        text = _("No popups"),
+        help_text = _("No popups are shown at all."),
+        checked_func = function()
+            return getMask() == Notification.SOURCE_NONE
+        end,
+        callback = function()
+            setMask(Notification.SOURCE_NONE)
+        end,
+        separator = true,
+        },
+        {
+        text = _("Only some bottom menu popus"),
+        help_text = _("Only show popups for bottom menu settings with no visual feedback."),
+        checked_func = function()
+            return band(getMask(), Notification.SOURCE_BOTTOM_MENU) == band(Notification.SOURCE_SOME, Notification.SOURCE_BOTTOM_MENU)
+        end,
+        callback = function()
+            if getMask() == Notification.SOURCE_ALL then
+                setMask(Notification.SOURCE_NONE)
+            end
+            setMask(bor(
+                    band(Notification.SOURCE_SOME, Notification.SOURCE_BOTTOM_MENU),
+                    band(getMask(), Notification.SOURCE_DISPATCHER)))
+        end,
+        },
+        {
+        text = _("A lot of bottom menu popus"),
+        help_text = _("Show popups from the bottom menu, which don't have visual feedback and popups from the three dots menus."),
+        checked_func = function()
+            return band(getMask(), Notification.SOURCE_BOTTOM_MENU) == band(Notification.SOURCE_DEFAULT, Notification.SOURCE_BOTTOM_MENU)
+        end,
+        callback = function()
+            if getMask() == Notification.SOURCE_ALL then
+                setMask(Notification.SOURCE_NONE)
+            end
+            setMask(bor(
+                    band(Notification.SOURCE_DEFAULT, Notification.SOURCE_BOTTOM_MENU),
+                    band(getMask(), Notification.SOURCE_DISPATCHER)))
+        end,
+        },
+        {
+        text = _("Gestures and profiles"),
+        help_text = _("Show popups on gestures and profiles."),
```

Comment:
```
Notifications from gestures and profiles
Show notification popups for changes from gestures and the profiles plugin.
```
Gestures is obvious. "Profiles" a bit less, dunno if it needs more explanations.